Classes/Queue/FileQueue.php

```diff
@@ -5,15 +5,19 @@
 namespace Pluswerk\Sentry\Queue;
 
 use Exception;
+use FilesystemIterator;
 use TYPO3\CMS\Core\Core\Environment;
 use TYPO3\CMS\Core\Utility\GeneralUtility;
 
 class FileQueue implements QueueInterface
 {
     private string $directory;
-    public function __construct()
+    private int $limit;
+    private bool $compress = true;
+    public function __construct(int $limit, bool $compress = true)
```

Comment:
i'd give the $limit a default-value, because at the moment this would break working instances on update

Reply:
nevermind, i've overseen the yaml-file